src/applications/gi/sass/partials/_gi-modal.scss

```diff
@@ -3,5 +3,8 @@
     .va-modal-inner {
       max-width: 600px;
     }
+    .va-modal-header-padding-top-30 {
```

Comment:
We should use the design system utilities and avoid adding application specific CSS when possible:

Looks like `.vads-u-padding-top--4` might be good.

 https://design.va.gov/utilities/padding#padding-top

Reply:
Okay, I'll go ahead and make the change.